lib/Dialect/TritonGPU/Transforms/WSCodePartition.cpp

```diff
@@ -927,15 +1038,13 @@ DenseMap<Channel *, Value> createBuffer(const SmallVector<Channel *> &channels,
       Type memdescType =
           tt::MemDescType::get(bufferShape, elemType, sharedLayout,
                                sharedMemorySpace, /*mutableMemory*/ true);
-      Value buffer;
-      if (isa<tt::LoadOp, tt::ExperimentalDescriptorLoadOp>(c->srcOp)) {
-        buffer =
-            builder.create<ttg::LocalAllocOp>(funcOp.getLoc(), memdescType);
-      } else {
-        buffer = builder.create<ttg::LocalAllocOp>(funcOp.getLoc(), memdescType,
-                                                   c->srcOperand);
```

Comment:
Yeah I was not sure why we need this else branch.

Reply:
Yeah seems unnecessary.